plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java

```diff
@@ -1141,11 +1152,13 @@ protected void selectPath( Object selectedElement, boolean useCache ) {
 
     } else if ( selectedElement instanceof Directory ) {
       try {
-        fileTableViewer.setInput( FILE_CONTROLLER.getFiles( (File) selectedElement, null, useCache ).stream().sorted(
-            Comparator.comparing( f -> f instanceof Directory, Boolean::compare ).reversed()
-              .thenComparing( Comparator.comparing( f -> ( (File) f ).getName(),
-                String.CASE_INSENSITIVE_ORDER ) ) )
-          .toArray() );
+        String searchString = txtSearch.getText();
```

Comment:
@addagudi Please see instructions in the README file in https://github.com/pentaho/pentaho-coding-standards to add checkstyle to your IDE. After implementing the checkstyle, please run it against this file to reformat your changes. Thank you.

Reply:
Fixed.